name-to-sid.py

```diff
@@ -36,13 +36,22 @@ def byteToLong(byte, little_endian=True):
             a = string.rjust(byte, 8, '\x00')
             return struct.unpack('>q', a)[0]
 
-def readLdapCredentials():
+def read_ldap_credentials():
+    """
+
+    """
     with open("/etc/secrets") as f:
         for line in f:
             if line.startswith('LDAP_PASSWORD='):
-                password = line[line.index('=') + 1:]
+                ldap_password = line[line.index('=') + 1:].strip()
+            elif line.startswith('LDAP_USER='):
+                ldap_user = eval(line[line.index('=') + 1:].strip())
```

Comment:
@deanlinssen Hey! By chance I saw this commit put up in my notifications. You **sure** you want to use `eval()` here??